app/Console/Commands/UpdateAtcActiveStatus.php

```diff
@@ -60,13 +60,14 @@ public function handle()
         $this->qualification_period = Setting::get('atcActivityQualificationPeriod', 12);
         $this->grace_period = Setting::get('atcActivityGracePeriod', 12);
         $this->hour_requirement = Setting::get('atcActivityRequirement', 10);
+        $handover_prefix = strlen(getenv('DB_HANDOVER_TABLE_PREFIX')) > 0 ? getenv('DB_HANDOVER_TABLE_PREFIX') : Config::get('database.connections.mysql-handover.prefix'); //Defaults to the prefix set in the DB config.
```

Comment:
There's not much reason for this line here, you can just inline the variable with `Config::get('database.connections.mysql-handover.prefix')`.
If there's no prefix, it will get an empty string from the config. If there is a prefix that would also be returned by `Config::get()`.

Reply:
As I was relooking over my PR, I checked the function and saw that it had a default-to prefix parameter defined. I'll keep that in mind next time, thanks!

Reply:
I'd prefer this to be resolved already now, so we get it right from the start :) Otherwise I'm sure we'll forget to clean this up later on.